_content/articles/hoard-of-inscribed-gandharan-metalware_salomon_richard.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "A Hoard of Inscribed Gandharan Metalware"
+authors:
+  - "Richard Salomon"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3291581&url=article"
+drive_links:
+  - "https://drive.google.com/file/d/15iT7pBJA3fO7dGUrfuC7Ve3MPa_yGagk/view?usp=sharing"
+course: medieval
```

Comment:
🧐  Maybe this is a bit early for "medieval"? Perhaps should be moved to "sects"?

Reply:
agreed